### PR TITLE
Support alternative receiver feature in Monitor

### PR DESCRIPTION
**Pull request by patitonar**
Built on top of https://github.com/poanetwork/tokenbridge/pull/221

The main changes to the monitor component were made on this commit: https://github.com/poanetwork/tokenbridge/commit/eef52e18a5e6e6ce15d92528ee060cdc7722aa14

Included also a fix for the chromedriver used in e2e tests to avoid CI failing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "initialize": "yarn clean && git submodule update --init && yarn install --unsafe-perm --frozen-lockfile && yarn install:deploy && yarn compile:contracts",
     "build": "yarn workspace ui run build",
     "lint": "yarn wsrun --exclude token-bridge-contracts lint",
-    "test": "yarn wsrun --exclude monitor --exclude oracle-e2e --exclude ui-e2e --exclude monitor-e2e test",
```
Comment (akolotov) [inline package.json]: why was monitor excluded before?

Comment (patitonar) [reply package.json]: The `test` command is to run the unit tests. The monitor has e2e tests but it didn't have unit tests so far, that's why it was excluded. I added the firsts unit tests for some methods here  https://github.com/poanetwork/tokenbridge/pull/223/commits/a0d6f20ae0cbd74c951af955c4428a16e86fed29

Comment (akolotov) [reply package.json]: thanks for clarification

**Comment by patitonar** [comment]
Yes, let's wait for https://github.com/poanetwork/tokenbridge/pull/221 to be merged in case more changes are applied there. Then we can update this branch with develop and merge this changes

**Comment by patitonar** [comment]
This is ready to be merged

**Pull request merged**